packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -255,6 +254,18 @@ export class PKPBase<T = PKPBaseDefaultParams> {
     }
   }
 
+  public async authenticate(): Promise<void> {
```

Comment:
copy/paste this if you need:
```
  /**
   * Authenticates the user - generate sessionSigs from litNodeClient(authContext.client) is provided. Otherwise, use
   * controllerSessionSigs.
   *
   * @returns A Promise that resolves to void.
   * @throws An error if `controllerSessionSigs` is not found.
   */
```